lib/src/clojure_lsp/feature/completion.clj

```diff
@@ -18,6 +18,11 @@
 
 (set! *warn-on-reflection* true)
 
+(def completion-kind-enum
+  {:text 1 :method 2 :function 3 :constructor 4 :field 5 :variable 6 :class 7 :interface 8 :module 9
+   :property 10 :unit 11 :value 12 :enum 13 :keyword 14 :snippet 15 :color 16 :file 17 :reference 18
+   :folder 19 :enummember 20 :constant 21 :struct 22 :event 23 :operator 24 :typeparameter 25})
```

Comment:
Not sure it's a good idea having this enum here as it's related to the LSP enum which is specific for `cli`, but I can't think on a better solution now

Reply:
Maybe it's part of fixing the borders to correctly coerce java->clj and clj->java